File: auto-docs/executables/python/Picnic_heatmap.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')

import numpy as np

size = 50
z = np.zeros((size, size)).tolist()
for r in range(1, size + 1):
    for c in range(1, size + 1):
        z[r-1][c-1] = np.sqrt(r*c/float(size**2))
data = Data([
    Heatmap(
        z=z,
        scl='Picnic'
    )
])
layout = Layout(
    title='Picnic'
)
fig = Figure(data=data, layout=layout)

plot_url = py.plot(fig, filename='Picnic-heatmap', auto_open=False)

File: auto-docs/executables/julia/Picnic_heatmap.jl
using Plotly

using Plotly
Plotly.signin("TestBot", "r1neazxo9w")

size = 50
z = rand(size, size)
for r = 1:size
  for c = 1:size
        z(r,c) = sqrt(r*c/size^2)
data = [
  [
    "z" => z, 
    "scl" => "Picnic", 
    "type" => "heatmap"
  ]
]
layout = ["title" => "Picnic"]

response = Plotly.plot([data], ["layout" => layout, "filename" => "Picnic-heatmap", "fileopt" => "overwrite", "auto_open" => "false"])
plot_url = response["url"]